meetings/2017-08-30.md

```diff
@@ -0,0 +1,107 @@
+None present
+
+## Agenda
+
+* announcements
+* review previous meeting
+
+### nodejs/node
+
+* promises: more robust stringification [#13784](https://github.com/nodejs/node/pull/13784)
+* stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+
+### nodejs/vm
+
+* drop --napi-modules earlier than dropping the "experimental" status [#9](https://github.com/nodejs/vm/issues/9)
+
+### nodejs/CTC
+
+* Requesting Rod to step down or be removed from the CTC [#165](https://github.com/nodejs/CTC/issues/165)
+
+
+## Previous Meeting Review
+
+We skipped this as all of the issues were on the agenda for this meeting as well.
+
+### nodejs/node
+
+* promises: more robust stringification [#13784](https://github.com/nodejs/node/pull/13784)
+* stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+* Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+
+### nodejs/vm
+
+* drop --napi-modules earlier than dropping the "experimental" status [#9](https://github.com/nodejs/vm/issues/9)
+
+
+## Minutes
+
+### Requesting Rod to step down or be removed from the CTC [#165](https://github.com/nodejs/CTC/issues/165)
+
+We agreed that initial discussion would best be handled through a medium other
+than email or voice discussion where it is hard to keep track of and
+collaborate on options. Nikita will send out a link to a doc to TSC/CTC
+members with data/initial suggestions.  It was discussed that we need to make
+sure internal discussion does not stretch out too long.  This discussion will
+also factor in the request in https://github.com/nodejs/board/issues/67
+within the board repo.
+
+
+### promises: more robust stringification [#13784](https://github.com/nodejs/node/pull/13784)
+
+Bring back to GitHub for further discussion.
+
+### stream: remove {writeableState/readableState}.length [#12857](https://github.com/nodejs/node/pull/12857)
+### Cleanup `_writableState` and `_readableState` access across codebase [#445](https://github.com/nodejs/node/issues/445)
+
+Skip these items because interested people are not present.
+
+### drop --napi-modules earlier than dropping the "experimental" status [#9](https://github.com/nodejs/vm/issues/9)
+
+* A PR was open: https://github.com/nodejs/node/pull/14902.
+  Objections should be made there.
+
+* Question is: do we remove the flag while keeping the feature experimental?
+  * Jeremiah: it might be better to keep the flag if a lot of breaking
+    changes are still expected.
+  * Michael: There are only two planned breaking changes and we would
+    remove the flag after they land.
+  * Ali: removing the flag will expose the feature more to the community.
+  * Left is as generally we are ok if flag is removed after any remaining
```

Comment:
I'm having trouble parsing this sentences. Is it supposed to be "Left as is" instead of "Left is as"?